app/perfsymboltable.cpp

```diff
@@ -328,7 +328,7 @@ int PerfSymbolTable::insertSubprogram(CuDieRangeMapping *cudie, Dwarf_Die *top,
     const QByteArray file = dwarf_decl_file(top);
 
     qint32 fileId = m_unwind->resolveString(file);
-    int locationId = m_unwind->resolveLocation(PerfUnwind::Location(entry, fileId, m_pid, line,
+    int locationId = m_unwind->resolveLocation(PerfUnwind::Location(entry, 0, fileId, m_pid, line,
```

Comment:
I cannot comment on the commit message in github, so I'm adding this here again. Note that this is a general remark that applies to most of your commit messages:

You are only saying what you are doing. But you do not explain _why_ you are doing it. In this case for example, the commit message should say something like

> A disassembler can then use the the relative address and size of a symbol to find the instructions that belong to a function.

Reply:
also, this patch and the previous one should already expand the unit test coverage by also printing the new values in the test client. Then update the expected files to show that this is working as intended (and doesn't break in the future).